locales/en/app.ftl

```diff
@@ -773,7 +773,20 @@ scan-results-known-breaches =
      *[other] This email appeared in { $breachCount } known data breaches.
   }
 
+# This string is shown at the top of the scan results page and is followed
+# by the email address that the user searched.
+# In page, it reads "Results for: searchedEmail@monitor.com"
+results-for = Results for
```

Comment:
Should this have a placeholder email variable in case certain locales need to control order?
/cc @flodolo

Reply:
The example in the comment is different: why there's a colon there, and not in the string?

But yes, a placeholder is safer.